README.md

```diff
@@ -1,2 +1,94 @@
 # puppet-module-etcd
-Puppet module to manage etcd
+
+#### Table of Contents
+
+1. [Module Description](#module-description)
+1. [Setup - The basics of getting started with Etcd](#setup)
+    * [What etcd affects](#what-etcd-affects)
+    * [Setup requirements](#setup-requirements)
+1. [Usage - Configuration options and additional functionality](#usage)
+    * [Basic Etcd](#basic-etcd)
+    * [Clustered Etcd](#clustered-etcd)
+    * [Etcd Upgrades](#etcd-upgrades)
+1. [Reference](#reference)
+1. [Limitations - OS compatibility, etc.](#limitations)
+
+## Module Description
+
+Installs and manages [Etcd](https://etcd.io/)
+
+### Documented with Puppet Strings
+
+[Puppet Strings documentation](http://tailored-automation.github.io/puppet-module-etcd/)
+
+## Setup - The basics of getting started with Etcd
+
+## What etcd affects
+
+This module will download the compiled binaries for etcd and extra the archive and install the necessary binaries, configuration files and services.
+
+## Setup requirements
+
+All module dependencies are listed in this module's `metadata.json`.
+
+## Usage - Configuration options and additional functionality
+
+### Basic Etcd
+
+To install and run a single instance of etcd it's sufficient to just include the `etcd` class:
+
+```puppet
+include etcd
+```
+
+All configuration is done via the `config` parameter, example:
```

Comment:
All configuration for the etcd.yaml, not ALL configs

Reply:
This is fixed.